runtime/altair/src/lib.rs

```diff
@@ -209,7 +209,7 @@ impl pallet_timestamp::Config for Runtime {
 
 // money stuff
 parameter_types! {
-	/// TransactionByteFee is set to 0.01 MicroAIR
```

Comment:
The reference to the token in the Altair runtime is AIR, so prob leave it as is.